### PR TITLE
Followers server

### PR DESCRIPTION
**Pull request by dcadenas**
I basically used the reportinator_server playbook with some differences. The main one is that the compose template it's also creating a neo4j service. I'm running it locally and we'll see if I'm missing something.

I know @gergelypolonkai is working in a separate neo4j playbook so this is still a draft so I later integrate it. I'm running the playbook anyways for my dev machine

### CODE REVIEW

File: inventories/nostr-rs-relay/inventory.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ relay:
   hosts:
     # We don't have any production servers using this role right now
     # The olympics relay should be compatible but there's a risk is not
-    104.236.4.237: # Daniel's server
+    123.123.123.123: # Used to be daniel's server, now followers server
```
Comment (gergelypolonkai) [inline inventories/nostr-rs-relay/inventory.yml:6]: This IP address is really suspicious to me.

**Review by gergelypolonkai — COMMENTED** [review]
Other than that strange IP address (123.123.123.123 is in China 😉), it looks good to me.

**Review by mplorentz — APPROVED** [review]
LGTM! Did you use this playbook to set up the production service? If so we should have @gergelypolonkai run it as well to make sure it works for him and then merge.

File: inventories/nostr-rs-relay/inventory.yml (hunk 1 above)
Comment (dcadenas) [reply inventories/nostr-rs-relay/inventory.yml:6]: oops, that was meant to be left as an example, there are no relays right now using that, I'm commenting it now

**Comment by dcadenas** [comment]
@mplorentz that's right so I'll wait till @gergelypolonkai checks that and then merge

**Pull request merged**